R-package/R/lgb.Dataset.R

```diff
@@ -277,7 +277,7 @@ Dataset <- R6::R6Class(
 
         # Setup initial scores
         init_score <- private$predictor$predict(
-          private$raw_data
+          data = private$raw_data
```

Comment:
this is correct! Just pointing out since you weren't sure, this is an instance of `Predictor` and comes from https://github.com/microsoft/LightGBM/blob/efa9ecf800e748d78c535d1e39588b4c0c419401/R-package/R/lgb.Predictor.R#L77